extensions/amp-analytics/0.1/requests.js

```diff
@@ -50,6 +50,9 @@ export class RequestHandler {
     /** @const {string} */
     this.baseUrl = dev().assert(request['baseUrl']);
 
+    /** @private {string} */
+    this.requestBody_ = JSON.stringify(request['body']) || '';
```

Comment:
a bit worried that do replacement on stringified result could cause problems. for example, do we want to allow variable/macro in the property names?